subsections/h3-shebangs.inc

```diff
@@ -1,16 +1,31 @@
 Are shebangs dragging you down (to Python 2)?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A shebang is an indicator on the first line of an executable script that indicates in what interpreter is the script supposed to be launched, examples include *python, bash and perl*. **If the shebang of the executable(s) in our package is pointing to Python 2 it needs to be updated to Python 3.**
+A shebang is an indicator on the first line of an executable script that indicates in what interpreter is the script supposed to be launched, examples include *python, bash and perl*. When software gets ported to Python 3, the lone shebang often remains forgotten and keeps pointing to Python 2. In most cases this is handled automatically: the ``setup.py`` script (usually run through the ``%py3_build`` and ``%py3_install`` RPM macros) adjusts shebangs for you. However, sometimes it's up to you to handle the situation.
 
-First find out if and what shebangs are used in your package by unpacking the sources for the project, ``cd``-ing into the unpacked directory and trying the following command(s):
+RPM has very good capabilities of automatically finding dependencies, and one of the ways it accomplishes that is by looking at the shebangs of all the files in the package. Therefore **it is important to check if the shebangs are not dragging in a runtime dependency on Python 2**.
+
+As the porting of the spec file is nearly finished, build it and then run the following analysis on the resulting Python 3 RPM file:
 
 .. code-block:: console
 
-   # Searches for all shebangs among the sources
+   $ # You can run this in mock as both rpm and grep are already installed.
```

Comment:
Because it's easier than building in koji and manually downloading the rpms, and potentially installing them to test them and messing your system with tons of potential dependencies.

In mock you can let a script do everything for you. At least that's why I use mock.